modules/hcp-update-using-mce-console.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-updating.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-update-using-mce-console_{context}"]
+= Updating a hosted cluster by using the {mce-short} console
+
+You can update your hosted cluster by using the {mce-short} console.
+
+.Procedure
+
+. Select *All clusters*.
+
+. Navigate to *Infrastructure* -> *Clusters* to view managed hosted clusters.
+
+. Click the *Upgrade available* link to update the control plane and node pools.
+
+The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
```

Comment:
Are we saying that the list of available updates should be checked from the following fields below, or that they are different from the actual list of available updates? It sounds a bit off to me.

Reply:
This is the current limitation and yes we want users to check the hosted cluster status. The HC update UI will have a link to this doc.